images/ose-baremetal-cluster-api-controllers.yml

```diff
@@ -0,0 +1,25 @@
+content:
+  source:
+    dockerfile: Dockerfile.ocp
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/cluster-api-provider-metal3.git
+      web: https://github.com/openshift/cluster-api-provider-metal3
+    ci_alignment:
+      streams_prs:
+        ci_build_root:
+          stream: rhel-9-golang-ci-build-root
+distgit:
+  branch: rhaos-{MAJOR}.{MINOR}-rhel-9
+enabled_repos:
+- rhel-9-baseos-rpms
+- rhel-9-appstream-rpms
+for_payload: true
+from:
+  builder:
+  - stream: rhel-9-golang
+  member: openshift-enterprise-base
+name: openshift/ose-baremetal-cluster-api-controllers
+owners:
+- kni-devel-deployment@redhat.com
```

Comment:
not metal-platform@redhat.com?

Reply:
Done with https://github.com/openshift-eng/ocp-build-data/pull/3586/commits/aa3c90cdd4aff54f46fa561641a6d7e263b2ab41